### PR TITLE
[App]  Navigation

### PR DESCRIPTION
**Pull request by whattablackhole**

### CODE REVIEW

File: rslang/src/app/navigation/dot-menu/dot-menu.component.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-dot-menu',
+  templateUrl: './dot-menu.component.html',
+  styleUrls: ['./dot-menu.component.scss'],
```
Comment (Anntol) [inline rslang/src/app/navigation/dot-menu/dot-menu.component.ts]: Do we need empty style file?

File: rslang/src/app/navigation/dot-menu/dot-menu.component.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      <mat-icon>games</mat-icon>
+      <span>Мини-игры</span>
+    </button>
+    <button
+      mat-menu-item
+      [routerLink]="['statistics']"
+      routerLinkActive="active"
+    >
+      <mat-icon>graphic_eq</mat-icon>
+      <span> Статистика</span>
```
Comment (Nikitzu) [inline rslang/src/app/navigation/dot-menu/dot-menu.component.html]: space

File: rslang/src/app/navigation/dot-menu/dot-menu.component.ts (hunk 1 above)
Comment (Nikitzu) [reply rslang/src/app/navigation/dot-menu/dot-menu.component.ts]: No

File: rslang/src/app/statistics/components/statistics-page/statistics-page.component.ts [hunk 3]
```diff
@@ -5,5 +5,4 @@ import { Component } from '@angular/core';
   templateUrl: './statistics-page.component.html',
   styleUrls: ['./statistics-page.component.scss'],
 })
-export class StatisticsPageComponent {
-}
+export class StatisticsPageComponent {}
```
Comment (Nikitzu) [inline rslang/src/app/statistics/components/statistics-page/statistics-page.component.ts]: Why all component classes in this PR named with word COMPONENT?

**Review by Nikitzu — CHANGES_REQUESTED** [review]

**Review by Nikitzu — APPROVED** [review]

**Review by Nikitzu — APPROVED** [review]

**Pull request merged**